src/util/debug.h

```diff
@@ -181,6 +174,18 @@ void sss_debug_fn(const char *file,
 
 #define APPEND_LINE_FEED 0x1 /* can be used as a sss_vdebug_fn() flag */
 
+/* Checks whether level is set in generic debug_level.
+   Rarely needed to be used explicitly as everything
+   should go to backtrace buffer anyway (regardless debug_level)
+   Deciding if "--verbose" should be passed to `adcli` child process
+   is one of usage examples.
+ */
+#define DEBUG_IS_SET(level) (debug_level & (level) || \
```

Comment:
This macro is still used in few places limiting debug output:
https://github.com/SSSD/sssd/blob/da55e3e69707de416b7949d08c165c950090bbb6/src/providers/ldap/sdap_async_initgroups_ad.c#L1425-L1439
https://github.com/SSSD/sssd/blob/da55e3e69707de416b7949d08c165c950090bbb6/src/providers/ldap/sdap_async_initgroups_ad.c#L1448-L1461
https://github.com/SSSD/sssd/blob/da55e3e69707de416b7949d08c165c950090bbb6/src/p11_child/p11_child_openssl.c#L1131-L1133

Reply:
It is intentional.
`get_ec_curve_type()` requires real communication with a smart card, this can be slow.
member/etc listing can be too large and would eat backtrace buffer.

I think it's ok to log this only in case `SSSDBG_TRACE_ALL` is set explicitly.